src/MLOps.NET.SQLite/Storage/LocalFileModelRepository.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace MLOps.NET.Storage
+{
+    internal sealed class LocalFileModelRepository : IModelRepository
+    {
+        public async Task UploadModelAsync(Guid runId, string filePath)
+        {
+            var modelStoragePath = @"C:\MLOps";
```

Comment:
@dcostea could we make this a variable passed into UseSQLite? I envision that one may want to store this on e.g. a fileshare or something else, so it would be awesome if it was configurable (we can let it default to C:\MLOps though so you don't have to set it)

Reply:
it makes sense